api/admin.py

```diff
@@ -168,3 +172,33 @@ class VidAdmin(admin.ModelAdmin):
 class InfoAdmin(admin.ModelAdmin):
     icon = '<i class="material-icons">link</i>'
     list_display = ('info_url', 'launch')
+
+
+@admin.register(models.AstronautStatus)
+class AstronautStatusAdmin(admin.ModelAdmin):
+    list_display = ('name',)
+
+
+@admin.register(models.Astronauts)
+class AstronautsAdmin(admin.ModelAdmin):
+    list_display = ('name', 'nationality')
+
+
+@admin.register(models.SpaceStationStatus)
+class SpaceStationStatusAdmin(admin.ModelAdmin):
+    list_display = ('name',)
+
+
+@admin.register(models.SpaceStation)
+class SpaceStationAdmin(admin.ModelAdmin):
+    list_display = ('name', )
+
+
+@admin.register(models.OrbiterStatus)
+class OrbiterStatusAdmin(admin.ModelAdmin):
```

Comment:
This was moved to configurations, right?

Reply:
Yes woops! Looks like I left the admin settings behind!